Rubberduck.Parsing/SelectionExtensions.cs

```diff
@@ -20,10 +19,12 @@ public static class SelectionExtensions
         public static bool Contains(this Selection selection, IToken token)
         {
             return
-                (((selection.StartLine == token.Line) && (selection.StartColumn - 1) <= token.Column) 
-                    || (selection.StartLine < token.Line))
-             && (((selection.EndLine == token.EndLine()) && (selection.EndColumn - 1) >= (token.EndColumn())) 
-                    || (selection.EndLine > token.EndLine()));
+                (selection.StartLine == token.Line
+                    && selection.StartColumn - 1 <= token.Column
+                    || selection.StartLine < token.Line)
+             && (selection.EndLine == token.EndLine()
+                    && selection.EndColumn - 1 >= token.EndColumn()
+                    || selection.EndLine > token.EndLine());
```

Comment:
I like the old linebreaking better, since it enhances semantic meaning ...

Reply:
obtw `- 1 >= ` is the same as `>` the difference in 1-based and 0-based should be explicitly mentioned again, though, which is why this is fine. A comment helps to not stumble over it every time...